From 4a8ccfdf57fb0e0835faef9d95939d31546202f8 Mon Sep 17 00:00:00 2001
From: Auger Eric <eric.auger@redhat.com>
Date: Wed, 19 Aug 2020 09:16:03 -0400
Subject: [PATCH 3/4] Disable TPM passthrough backend on ARM
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Auger Eric <eric.auger@redhat.com>
Message-id: <20200819091603.23319-1-eric.auger@redhat.com>
Patchwork-id: 98190
O-Subject: [RHEL-AV-8.3.0 qemu-kvm PATCH] Disable TPM passthrough backend on ARM
Bugzilla: 1801242
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>
RH-Acked-by: Andrew Jones <drjones@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1801242
UPSTREAM: not applicable
BRANCH: rhel-av-8.3.0
BREW: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=30820412

Let's disable the TPM passthrough backend on ARM as this looks
to be a marginal use case and it was not tested comprehensively yet.

Signed-off-by: Eric Auger <eric.auger@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 default-configs/aarch64-rh-devices.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/default-configs/aarch64-rh-devices.mak b/default-configs/aarch64-rh-devices.mak
index 0b6a03f816..6a597384ea 100644
--- a/default-configs/aarch64-rh-devices.mak
+++ b/default-configs/aarch64-rh-devices.mak
@@ -21,5 +21,4 @@ CONFIG_XIO3130=y
 CONFIG_NVDIMM=y
 CONFIG_ACPI_APEI=y
 CONFIG_TPM_EMULATOR=y
-CONFIG_TPM_PASSTHROUGH=y
 CONFIG_TPM_TIS_SYSBUS=y
-- 
2.27.0

